server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestCancelTasksAction.java

```diff
@@ -66,6 +66,8 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         cancelTasksRequest.setNodes(nodesIds);
         cancelTasksRequest.setActions(actions);
         cancelTasksRequest.setParentTaskId(parentTaskId);
+        cancelTasksRequest.setWaitForCompletion(
+            request.paramAsBoolean("wait_for_child_tasks", CancelTasksRequest.DEFAULT_WAIT_FOR_COMPLETION));
```

Comment:
instead of introducing this constant, just use `cancelTasksRequest.getWaitForCompletion()` here